tests/ui/lint/const-item-interior-mutations-const-atomics.stderr

```diff
@@ -0,0 +1,816 @@
+warning: mutation of an interior mutable `const` item with call to `load`
+  --> $DIR/const-item-interior-mutations-const-atomics.rs:13:14
+   |
+LL |     let _a = A.load(Ordering::SeqCst);
+   |              -^^^^^^^^^^^^^^^^^^^^^^^
+   |              |
+   |              `A` is a interior mutable `const` item of type `AtomicBool`
+   |
+   = note: each usage of a `const` item creates a new temporary
+   = note: only the temporaries and never the original `const A` will be modified
+   = help: for more details on interior mutability see <https://doc.rust-lang.org/reference/interior-mutability.html>
+   = note: `#[warn(const_item_interior_mutations)]` on by default
+help: for a shared instance of `A`, consider making it a `static` item instead
+   |
+LL -     const A: AtomicBool = AtomicBool::new(false);
+LL +     static A: AtomicBool = AtomicBool::new(false);
```

Comment:
It's definitely a bit strange to call load on a const atomic, but I think the lint would be confusing because it is not "mutation of an interior mutable `const` item".

Specifically the sentence `mutation of an interior mutable const item with call to load` is not technically correct

I think either:
- The lint should not specifically accuse of "mutation"
- We should remove this case